builders/build/substrate/adding-built-in-pallet.md

```diff
@@ -0,0 +1,151 @@
+
+### Declare the dependency {: #declare-dependency }
+
+Every package contains a manifest file named `Cargo.toml` stating, among other things, all the dependencies the package relies on, and the ContainerChain runtime is no exception. 
+
+To declare the dependency and make it available to the runtime, open the `Cargo.toml` file located in the folder `runtime` with a text editor and add the module, referencing the code in the official repository of the Polkadot SDK:
+
+```toml
+[dependencies]
+...
+pallet-assets = { git = "https://github.com/paritytech/polkadot-sdk", branch = "master", default-features = false }
+...
+```
+
+### Make the standard features available to the compiler {: #standard-features }
+
+In the `Cargo.toml` file there is a features section where the features from the module marked as standard must be added, to make them available to the compiler to build the runtime binary:
+
+```toml
+[features]
+default = [
+	"std",
+]
+std = [
+	...,
+	"pallet-assets/std",
+   ...
+]
+```
+### Configure the Module {: #configure-the-module }
+
+With the dependency declared, now the module can be configured and added to the runtime to use it. It is done in the `lib.rs` file that is located in the folder */runtime/src*.
+
+The following code snippet is a basic example that configures the module with types, constants and default values. These values must be adjusted to the specific requirements of the use case.
+
+```rust
+...
+parameter_types! {
+	pub const AssetDeposit: Balance = 100;
+	pub const ApprovalDeposit: Balance = 1;
+	pub const StringLimit: u32 = 50;
+	pub const MetadataDepositBase: Balance = 10;
+	pub const MetadataDepositPerByte: Balance = 1;
+}
+
+impl pallet_assets::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Balance = u128;
+	type AssetId = u64;
+	type AssetIdParameter = u64;
+	type Currency = Balances;
+	type CreateOrigin = frame_support::traits::AsEnsureOriginWithArg<frame_system::EnsureSigned<AccountId>>;
+	type ForceOrigin = EnsureRoot<AccountId>;
+	type AssetDeposit = AssetDeposit;
+	type AssetAccountDeposit = frame_support::traits::ConstU128<1>;
+	type MetadataDepositBase = MetadataDepositBase;
+	type MetadataDepositPerByte = MetadataDepositPerByte;
+	type ApprovalDeposit = ApprovalDeposit;
+	type StringLimit = StringLimit;
+	type Freezer = ();
+	type Extra = ();
+	type WeightInfo = pallet_assets::weights::SubstrateWeight<Runtime>;
+	type RemoveItemsLimit = frame_support::traits::ConstU32<1000>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkHelper = ();
+   type CallbackHandle = ();
+}
+...
+```
+
+It is important to note that every built-in module has a different purpose, and therefore, have different needs in term of what must be configured. 
```

Comment:
I think it is important that you explain (at least as an example) what some of this configuration parameters are. Also you never explain what are the `parameter_types!` and what are the `impl pallet_assets::Config for Runtime {` - Remember that you have to write this more or less from the angle of someone who is not really familiar with this

Reply:
I would explain what each thing is and then a few of the configurations